<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>425</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Agregar Roles</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">background : rgb(250,238,224)</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>423</width>
        <height>298</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QPushButton" name="btnNuevoRol">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background : rgb(174, 116, 0);
color : rgb(255, 255, 255)
}
QPushButton::disabled {
background : rgb(187, 129, 13);
color : rgb(245,245,245);
border: 1px solid #adb2b5;
}</string>
         </property>
         <property name="text">
          <string>Nuevo</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btnAceptar">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background : rgb(174, 116, 0);
color : rgb(255, 255, 255)
}
QPushButton::disabled {
background : rgb(187, 129, 13);
color : rgb(245,245,245);
border: 1px solid #adb2b5;
}</string>
         </property>
         <property name="text">
          <string>Aceptar</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="btnCancelar">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background : rgb(174, 116, 0);
color : rgb(255, 255, 255)
}
QPushButton::disabled {
background : rgb(187, 129, 13);
color : rgb(245,245,245);
border: 1px solid #adb2b5;
}</string>
         </property>
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QTableWidget" name="twRoles">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>387</width>
           <height>250</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableWidget {background-color: white;}
QHeaderView::section
{
background-color: rgb(238,225,200);
color : rgb(52, 42, 24 );
text-align: right;
font-size:12px;
height:24px
}
QHeaderView {background-color: white;}</string>
         </property>
         <column>
          <property name="text">
           <string>Rol</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="leBusqueda">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background : rgb(255,255,255)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>94</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Roles:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnNuevoRol</tabstop>
  <tabstop>leBusqueda</tabstop>
  <tabstop>btnAceptar</tabstop>
  <tabstop>btnCancelar</tabstop>
  <tabstop>twRoles</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
